### ISSUE TITLE
Confirm link screen lang is missing [b1.7.3]

### ISSUE THREAD

**Issue by kimoVoid** [OrnitheMC/modmenu#19]
The lang file for mcb1.7.3 (I assume other legacy versions as well) is missing these keys. The position/width of the buttons also needs fixing.
![{3FC348BF-7165-452C-8640-812BB327563B}](https://github.com/user-attachments/assets/a8e8f4fa-8c27-4246-9c75-49db11f58a14)

**Comment by SpaceWalkerRS** [comment]
IIRC Mod Menu uses vanilla translation strings. I suppose those specific ones did not exist yet in b1.7.3? We'll have to add our own lang file to add those translations then.

**Comment by kimoVoid** [comment]
Yup, they just gotta be added to the lang file. And the buttons have to be moved and resized to be centered, but I think that's also an issue on other versions of the mod.

**Comment by SpaceWalkerRS** [comment]
this issue affected versions up to 1.1 - it's now finally fixed!
